### ISSUE TITLE
EJB, @EJB and @Default

### ISSUE THREAD

**Issue by rene-anderes** [cdi-unit/cdi-unit#79]
Given (2 classes):

```
@Stateless
public class TestEjbSimple { ... }'
```

and

```
    @RunWith(CdiRunner.class)
    @AdditionalClasses({ TestEjbSimple.class })
    @SupportEjb
    public class TestEjb {
        @EJB
        private TestEjbSimple ejbSimple; 
        @Test
        public void testEjb() {
            assertNotNull(ejbSimple);
        }
    }
```

Result: The test is red.

Change the EJB:

```
@Default
@Stateless
public class TestEjbSimple { ... }
```

Result: The test is green!

IMHO the test would also be green without @Default, isn't it?

CDI-Unit 3.1.3, Java 8

**Comment by ikysil** [comment]
Dear @rene-anderes ,

The support for EJB injection had been significantly improved by PR #85.

Could you please check if your issue is solved by that PR?

**Comment by ikysil** [comment]
Closing as seems to be fixed by #85
